README.md

```diff
@@ -1,18 +1,27 @@
-# Краткая инструкция по работе
+# Work instructions 👩‍🎓👨‍🎓
+
+## To get started, must be installed: 🔔
 
-## Для начала работы должен быть установлен:
 - Node.js v.10.15.3
 - Gulp v4
 - npm last version
 
-## Основные команды для работы
-- Установка - npm i
-- Запуск локального сервера - npm start
-- Сборка проекта без запуска локального сервера - npm run build
-- Запуск тестирования на соответствия код-гайдам - npm test
+## Basic commands for work 💻
+
+- Installation - npm i
+- Running a local server - npm start
+- Running a project without starting a local server - npm run build
+- Running testing with code guides - npm test
+
+All development is done in the source directory.
 
-В папке package.json изменить имя и описание.
+The resulting code goes into the build directory.
 
-Все разработка ведётся в директории source.
+## Workflow ✅
```

Comment:
1. Emoji should be written with shortcode (https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md)